core/domain/exp_jobs_one_off.py

```diff
@@ -503,13 +506,20 @@ def map(item):
         for html_data in html_list:
             soup = bs4.BeautifulSoup(html_data, 'html.parser')
 
+            # Any html string with unwrapped text is also invalid.
```

Comment:
It may not be clear to others what "unwrapped text" is. Say something like "text with no parent tag".

Reply:
Done! Thanks.